### ISSUE TITLE
Connect jabref export layouts with latex

### ISSUE THREAD

**Issue by CesMak** [JabRef/layouts.jabref.org#3]
Hey there,

I have a lot of articles and books in my jabref and connected it with latex using:

\bibliography{./Literatur/lit}

In the Literatur folder I have 3 files: BibtexFile (lit.bib), lit.bib.bak, lit.bib.sav

Now I want to change my literature file. Currently all authors with all names and last names are included (in capital letters...). I read that I need custom exports, but I cannot export to a BibTex File using those. The main supported export layout format seems to be *.rtf which I cannot read in latex....

Is there any method to use layouts from this repro as export filters for latex?

**Comment by koppor** [comment]
No, this is not planned. Do you use bibtex or biblatex? I would recommend biblatex. See https://en.wikibooks.org/wiki/LaTeX/Bibliographies_with_biblatex_and_biber for details.

Please be aware that bibtex (or biber/biblatex) renders the output for latex, not jabref

![grafik](https://user-images.githubusercontent.com/1366654/82425960-86242680-9a87-11ea-923d-127e8b3c9b8f.png)

See http://www.suedraum.de/latex/stammtisch/degenkolb_latex_biblatex_folien.pdf

**Comment by CesMak** [comment]
Thanks for the hints :+1:  

Issue can be closed.

**Comment by koppor** [comment]
I leave it open as a reminder to refine the documentation.

This issue also refs https://github.com/koppor/jabref/issues/392
